content/blog/2022-07-26-iterative-studio-model-registry.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Git-backed Machine Learning Model Registry to bring order to chaos
+date: 2022-07-26
+description: >
+  🚀 As Machine Learning projects and teams grow, keeping track of all the
+  models and their production status gets increasingly complex. Iterative
+  Studio's Git-backed Model Registry solves this.
+descriptionLong: >
+  Use your Git repositories to build a model registry with model versioning,
+  lineage, and lifecycle management. With Iterative Studio, have the who, what,
+  why, where and when questions of your team's model production at your
+  fingertips. Read to find out how.
+picture: 2022-07-26/model-management-chaos.png
+pictureComment:
+  Without the right tools, model management can easily turn chaotic
+author: tapa_dipti_sitaula
+commentsUrl: https://discuss.dvc.org/t/serving-machine-learning-models-with-mlem/1253
+tags:
+  - Model Registry
+  - MLOps
+  - Git
+  - MLEM
+  - GTO
+  - Iterative Studio
+  - Release
+---
+
+Machine learning tasks are iterative by nature. Over time, you build several
+versions of your ML models, which could be in different stages of
+production-readiness. A version may be running in production, another version
+that seems to perform better may be in staging, and a couple more versions could
+be in active development by you and your teammates - using updated
+hyperparameters, datasets, or algorithms.
+
+How do you keep track of all your models, their versions, and deployment
+statuses? How do you get answers to questions like these easily:
+
+- Which model version is currently in production?
+- When was the last time this model was updated?
+
+If you are like some of the data scientists we know, you may have a Google sheet
+or a Notion page with the list of all your models, their changes, deployment
+history, and so on. But this is highly error-prone and will probably get
+out-of-date very quickly. Or maybe you upload all your models to a cloud bucket
+and “attach” text reports to them. Not very maintainable or searchable either.
+We’ve even seen people use sticky notes, or better yet, rely on their memory 😀
+
+Some of the more organized folks use Model Registries - tools created
+specifically to organize models into a central, searchable repository. While
+this is definitely better than using random files or sticky notes, one major
+problem persists: the data science and machine learning team members work
+completely isolated from the software development and DevOps team members. This
+makes collaboration far more time consuming than it should be.
+
+![Teams can work in disconnected silos](/uploads/images/2022-07-26/disconnected-silos.png)
```

Comment:
This photo isn't centered properly.  Looks awkward.  Let me know if you need me to correct it.

Reply:
Fixed.